doc/source/features/strict-mode.rst

```diff
@@ -0,0 +1,384 @@
+ a warning is logged and startup continues.
+
+ At runtime, it is permitted to change ``binlog_format``
+ to any value, but if you set it to anything other than ``ROW``,
+ a warning is logged.
+
+``ENFORCING`` or ``MASTER``
+
+ At startup, if ``binlog_format`` is set to anything other than ``ROW``,
+ an error is logged and startup is aborted.
+
+ At runtime, any attempt to change ``binlog_format``
+ to anything other than ``ROW`` fails and an error is logged.
+
+.. |binlog_format| replace:: ``binlog_format``
+.. _binlog_format: http://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_format
+
+Tables without primary keys
+---------------------------
+
+|PXC| cannot properly propagate certain write operations
+to tables that do not have primary keys defined.
+Undesirable operations include data manipulation statements
+that perform writing to table (especially ``DELETE``).
+
+Depending on the strict mode selected,
+the following happens:
+
+``DISABLED``
+
+ At startup, no validation is performed.
+
+ At runtime, all operations are permitted.
+
+``PERMISSIVE``
+
+ At startup, no validation is perfromed.
+
+ At runtime, all operations are permitted,
+ but a warning is logged when an undesirable operation
+ is performed on a table without an explicit primary key defined.
+
+``ENFORCING`` or ``MASTER``
+
+ At startup, no validation is performed.
+
+ At runtime, any undesirable operation
+ performed on a table without an explicit primary key
+ is denied and an error is logged.
+
+Log output
+----------
+
+|PXC| does not support tables in the MySQL database
+as the destination for log output.
+By default, log entries are written to file.
+This validation checks the value of the |log_output|_ variable.
+
+Depending on the strict mode selected,
+the following happens:
+
+``DISABLED``
+
+ At startup, no validation is performed.
+
+ At runtime, you can set ``log_output`` to any value.
+
+``PERMISSIVE``
+
+ At startup, if ``log_output`` is set only to ``TABLE``,
+ a warning is logged and startup continues.
+
+ At runtime, it is permitted to change ``log_output``
+ to any value, but if you set it only to ``TABLE``,
+ a warning is logged.
+
+``ENFORCING`` or ``MASTER``
+
+ At startup, if ``log_output`` is set only to ``TABLE``,
+ an error is logged and startup is aborted.
+
+ At runtime, any attempt to change ``log_output`` only to ``TABLE`` fails
+ and an error is logged.
+
+.. |log_output| replace:: ``log_output``
+.. _log_output: http://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_log_output
+
+.. _explicit-table-locking:
+
+Explicit table locking
+----------------------
+
+|PXC| has only experimental support for explicit table locking operations,
+The following undesirable operations lead to explicit table locking
+and are covered by this validation:
+
+* ``LOCK`` and ``UNLOCK``
+* ``GET_LOCK()`` and ``RELEASE_LOCK()``
+* ``FLUSH TABLE ... WITH READ LOCK``
```

Comment:
there are 2 flush table.

flush table <table> with read read lock; (this is blocked)

flush table with read lock; (this is allowed)

So just to be extra careful we should probably mention it as "FLUSH TABLE <table> WITH READ LOCK"